src/_locales/en/messages.json

```diff
@@ -196,6 +196,9 @@
   "connectWallet_error_grantRejected": {
     "message": "Wallet connection cancelled. The request was not authorized."
   },
+  "connectWallet_error_timeout": {
+    "message": "Wallet connection timed out."
```

Comment:
Need better message here @melissahenderson

Reply:
Shown in popup:

![image](https://github.com/user-attachments/assets/5faa1c07-b2c9-45cc-ba0a-fadaa9f7ce71)

Reply:
If anything maybe add "Please try again". I know the tab has something similar (please try reconnecting wallet), but the extension should have problem/resolution as well.

Reply:
```suggestion
    "message": "Connecting to wallet took too long. Please try again."
```